include/TimeSync.hpp

```diff
@@ -0,0 +1,70 @@
+#pragma once
+#include <thread>
+#include <spdlog/spdlog.h>
+
+#include <carma-clock/carma_clock.h>
+#include <rapidjson/document.h>
+#include <udp-socket/UdpServer.hpp>
+#include <atomic>
+
+namespace carma_streets_time_sync {
+    /**
+     * TimeSyncMessage struct
```

Comment:
It would be more helpful to include comment about the purpose of each fields in the struct?